test/services/fileStorage/aws/awsS3.test.js

```diff
@@ -4,6 +4,10 @@ const { expect } = require('chai');
 const mockery = require('mockery');
 const mockAws = require('./s3.mock');
 const logger = require('../../../../src/logger');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
for merge to develop/release:
```suggestion
const { Configuration } = require('@hpi-schul-cloud/commons');
```